docs/openapi.md

```diff
@@ -0,0 +1,13 @@
+## What is the Klaw API source of truth?
+
+The OpenAPI spec `./openapi.yaml` at the root of the project is our source of truth. It is _auto-generated_ from the current state of the API code, and is therefore always up to date with the current state of the API.
+
+We use this OpenAPI spec as a source for the generation of the TypeScript types the frontend relies on to accurately type our business logic (mainly in [the `.coral/domain` folder](../coral/docs/directory-structure.md#first-level-domain-folder)).
+
+Therefore, every time the API changes, the OpenAPI spec will change, and the generated types for the frontend will keep in sync with the evolution of the API.
+
+## Generating the OpenAPI spec and TypeScript types
+
+This is a high level diagram of the process described above.
+
+![Generation](./generation_schema.png)
```

Comment:
I think it would be super nice to have that diagram "translated" to text, too, in the `alt` text. Otherwise the information is lost for screenreader users (why only try to be accessible in the app 😇 ).

Reply:
The diagram is an illustration of the process described in the first section, and there is a sentence explaining what the diagram is before it. I do agree that the alt ext may be a little terse, so I made it more explicit. But I was afraid of being redundant :o

Reply:
[ff60b57](https://github.com/aiven/klaw/pull/884/commits/ff60b57c9ef6abf15bd59af5ca18d84daf513ae0)

Reply:
I think that's great now!